manifests/daemonset.yaml

```diff
@@ -68,7 +68,7 @@ spec:
W1wIG0taW5ncmVzcykgfHwgZXhpdFdpdGhFcnJvciAiRmFpbGVkIHRvIGFkZCBydWxlOiBqdW1wIG0taW5ncmVzcyIKCiAgICAgICAgIyBzZXR1cCBwb3N0cm91dGluZyBjaGFpbgogICAgICAgIGlwIG5ldG5zIGV4ZWMgIiRDTklfQ09OVEFJTkVSSUQiICQobmZ0X2FkZF9ydWxlICR7TkZUX1RBQkxFX0JSSURHRX0gJHtORlRfVEFCTEV9ICR7TkZUX1BPU1RST1VUSU5HX0NIQUlOfSBldGhlciB0eXBlIGFycCBjb3VudGVyIGFjY2VwdCkgfHwgZXhpdFdpdGhFcnJvciAiRmFpbGVkIHRvIGFsbG93IGFycCBmb3IgZWdyZXNzIgogICAgICAgIGlwIG5ldG5zIGV4ZWMgIiRDTklfQ09OVEFJTkVSSUQiICQobmZ0X2FkZF9ydWxlICR7TkZUX1RBQkxFX0JSSURHRX0gJHtORlRfVEFCTEV9ICR7TkZUX1BPU1RST1VUSU5HX0NIQUlOfSBvaWZuYW1lICRDTklfSUZOQU1FIGNvdW50ZXIganVtcCBtLWVncmVzcykgfHwgZXhpdFdpdGhFcnJvciAiRmFpbGVkIHRvIGFkZCBydWxlOiBqdW1wIG0tZWdyZXNzIgoKICAgICAgICBpcCBuZXRucyBleGVjICIkQ05JX0NPTlRBSU5FUklEIiAkKG5mdF9hZGRfcnVsZSAiJHtORlRfVEFCTEVfTkVUREVWfSIgIiR7TkZUX1RBQkxFfSIgbS1pbmdyZXNzIGNvdW50ZXIgbWV0YSBtYXJrIHNldCBtZXRhIG1hcmsgIiYiIDB4ZmZmY2ZmZmYpIHx8IGV4aXRXaXRoRXJyb3IgIkZhaWxlZCB0byBhZGQgcnVsZTogc2V0IG1ldGEgbWFyayAmIDB4ZmZmY2ZmZmYiCiAgICAgICAgaXAgbmV0bnMgZXhlYyAiJENOSV9DT05UQUlORVJJRCIgJChuZnRfYWRkX3J1bGUgIiR7TkZUX1RBQkxFX0JSSURHRX0iICIke05GVF9UQUJMRX0iIG0tZWdyZXNzIGNvdW50ZXIgbWV0YSBtYXJrIHNldCBtZXRhIG1hcmsgIiYiIDB4ZmZmY2ZmZmYpIHx8IGV4aXRXaXRoRXJyb3IgIkZhaWxlZCB0byBhZGQgcnVsZTogc2V0IG1ldGEgbWFyayAmIDB4ZmZmY2ZmZmYiCgogICAgICAgIGNyZWF0ZV9jaGFpbnNfZm9yX2ZpbHRlcmluZyAkcG9saWN5X2lkICR7TkZUX1RBQkxFX05FVERFVn0gImluZ3Jlc3MiCiAgICAgICAgY3JlYXRlX2NoYWluc19mb3JfZmlsdGVyaW5nICRwb2xpY3lfaWQgJHtORlRfVEFCTEVfQlJJREdFfSAiZWdyZXNzIgoKICAgICAgICBpcCBuZXRucyBleGVjICIkQ05JX0NPTlRBSU5FUklEIiAkKG5mdF9hZGRfcnVsZSAiJHtORlRfVEFCTEVfTkVUREVWfSIgIiR7TkZUX1RBQkxFfSIgbS1pbmdyZXNzIG1ldGEgbWFyayAiJiIgMHgwMDAzMDAwMCA9PSAweDAwMDMwMDAwIGNvdW50ZXIgcmV0dXJuKSB8fCBleGl0V2l0aEVycm9yICJGYWlsZWQgdG8gYWRkIHJ1bGU6IG1hcmsgJiAweDAwMDMwMDAwID09IDB4MDAwMzAwMDAgY291bnRlciByZXR1cm4iCiAgICAgICAgaXAgbmV0bnMgZXhlYyAiJENOSV9DT05UQUlORVJJRCIgJChuZnRfYWRkX3J1bGUgIiR7TkZUX1RBQkxFX05FVERFVn0iICIke05GVF9UQUJMRX0iIG0taW5ncmVzcyBjb3VudGVyIGRyb3ApIHx8IGV4aXRXaXRoRXJyb3IgIkZhaWxlZCB0byBhZGQgcnVsZTogZHJvcCIKCiAgICAgICAgaXAgbmV0bnMgZXhlYyAiJENOSV9DT05UQUlORVJJRCIgJChuZnRfYWRkX3J1bGUgIiR7TkZUX1RBQkxFX0JSSURHRX0iICIke05GVF9UQUJMRX0iIG0tZWdyZXNzIG1ldGEgbWFyayAiJiIgMHgwMDAzMDAwMCA9PSAweDAwMDMwMDAwIGNvdW50ZXIgcmV0dXJuKSB8fCBleGl0V2l0aEVycm9yICJGYWlsZWQgdG8gYWRkIHJ1bGU6IG1hcmsgJiAweDAwMDMwMDAwID09IDB4MDAwMzAwMDAgY291bnRlciByZXR1cm4iCiAgICAgICAgaXAgbmV0bnMgZXhlYyAiJENOSV9DT05UQUlORVJJRCIgJChuZnRfYWRkX3J1bGUgIiR7TkZUX1RBQkxFX0JSSURHRX0iICIke05GVF9UQUJMRX0iIG0tZWdyZXNzIGNvdW50ZXIgZHJvcCkgfHwgZXhpdFdpdGhFcnJvciAiRmFpbGVkIHRvIGFkZCBydWxlOiBkcm9wIgoKICAgICAgICBleGl0V2l0aFN1Y2Nlc3MKICAgIDs7CgogICAgREVMKQogICAgICAgIGVjaG8gIkRlbGV0ZSAkQ05JX0NPTlRBSU5FUklEIiA+PiAkbG9nRmlsZQogICAgICAgIHJtIC1mIC92YXIvcnVuL25ldG5zLyIkQ05JX0NPTlRBSU5FUklEIgogICAgOzsKCiAgICBWRVJTSU9OKQogICAgICAgIGVjaG8gInsiY25pVmVyc2lvbiI6IjAuMy4xIiwic3VwcG9ydGVkVmVyc2lvbnMiOlsiMC4xLjAiLCIwLjIuMCIsIjAuMy4wIiwiMC4zLjEiXX0iCiAgICA7OwoKICAgICopCiAgICAgICAgZXhpdFdpdGhFcnJvciAiVW5yZWNvZ25pemVkIENOSSBjb21tYW5kIgogICAgOzsKCiAgICBlc2FjCn0KCm1haW4K"
```

Comment:
is this updated automatically ? 
maybe worth to have a script that does it + CI job that checks it was updated ?
easy to forget otherwise

Reply:
Yep, we'll need to setup CI checking this, for now it's just a `make generate` target

Reply:
np, but just before its merged, lets have the latest base64 if possible